gson/src/test/java/com/google/gson/functional/GsonParsingTest.java

```diff
@@ -26,22 +26,20 @@
 import com.google.gson.common.TestTypes.BagOfPrimitives;
 import com.google.gson.common.TestTypes.Nested;
 import com.google.gson.reflect.TypeToken;
-
-import junit.framework.TestCase;
-
 import java.io.StringReader;
 import java.lang.reflect.Type;
 import java.util.Arrays;
 import java.util.List;
 import java.util.Map;
+import junit.framework.TestCase;
 
 /**
- * Functional tests for that use JsonParser and related Gson methods
+ * Functional tests for Gson parsing methods
  *
  * @author Inderjeet Singh
  * @author Joel Leitch
  */
-public class JsonParserTest extends TestCase {
+public class GsonParsingTest extends TestCase {
```

Comment:
Have renamed this class because there exists already a `JsonParserTest` (in a different package), and this test class is mainly checking the behavior of `Gson` methods.